src/mSupplyMobileApp.js

```diff
@@ -291,14 +284,7 @@ class MSupplyMobileAppContainer extends React.Component {
             <MainStackNavigator.Screen name={ROUTES.DASHBOARD} component={DashboardPage} />
           </MainStackNavigator.Navigator>
 
-          <FinaliseModal
-            database={UIDatabase}
-            isOpen={finaliseModalOpen}
-            onClose={closeFinaliseModal}
-            finaliseItem={finaliseItem}
-            user={currentUser}
-            runWithLoadingIndicator={this.runWithLoadingIndicator}
-          />
+          <FinaliseModal />
```

Comment:
user and finalise item are coming from redux. RunwithLoadingIndicator is now in a context and the open/close are dispatchers to redux. Database from the singleton